public/content/developers/tutorials/server-components/index.md

```diff
@@ -287,4 +287,4 @@ These are packages that are required at runtime, when running `dist/app.js`.
 
 The centralized server we created here does its job, which is to act as an agent for a user. Anybody else who wants the dapp to continue functioning and is willing to spend the gas can run a new instance of the server with their own address. 
 
-However, this only works when the centralized server's actions can be easily verified. If the centralized server has any secret state information, or runs difficult calculations, it is a centralized entity that you need trust to use the application, which is exactly what blockchains try to avoid. In a future article I plan to show how to use [zero-knowledge proofs](/zero-knowledge-proofs) to get around this problem.
\ No newline at end of file
+However, this only works when the centralized server's actions can be easily verified. If the centralized server has any secret state information, or runs difficult calculations, it is a centralized entity that you need trust to use the application, which is exactly what blockchains try to avoid. In a future article I plan to show how to use [zero-knowledge proofs](/zero-knowledge-proofs) to get around this problem.
```

Comment:
In case this is not clear, this adds a new line at the end of the file (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)